README.md

```diff
@@ -27,7 +27,7 @@ We handle the complexity of realtime messaging so you can focus on your code.
 
```

Comment:
@marklewin this may be a good starting point, but `Push Notifications.md` is the majority of the documentation.